src/main/java/appeng/integration/BaseModule.java

```diff
@@ -21,12 +21,6 @@
 
 public abstract class BaseModule implements IIntegrationModule
 {
-
-	protected void testClassExistence( Class<?> clz )
-	{
-		clz.isInstance( this );
-	}
-
 	@Override
 	public abstract void init() throws Throwable;
```

Comment:
I meant the `init()` and such methods, since they are defined as `public abstract`

The implementation is actually a no-brainer, since it is only used, if the definitions are existent. The semantic is non-existent, since the return value does not matter. You could actually remove the pointer to `this` everywhere and use some instance, like `clz.isInstance( clz )` also possible. We could make it explicit by defining the constructors to throw `NoClassBlubBla` exception or some some actual method like `validate() throws NoClassDefFoundException` since validation is not meant to be in the constructor(imo?)